flood_adapt/dbs_classes/dbs_tipping_point.py

```diff
@@ -0,0 +1,71 @@
+import shutil
+
+from flood_adapt.dbs_classes.dbs_template import DbsTemplate
+from flood_adapt.object_model.interface.tipping_points import ITipPoint
+from flood_adapt.object_model.tipping_point import TippingPoint
+
+
```

Comment:
I based on the other codes and have not added other functionalities. Maybe we need to add the plots here?